### ISSUE TITLE
[Feature Request] When adding PuP-Pack, as user if they want to reset

### ISSUE THREAD

**Issue by Ltek** [syd711/vpin-studio#673]
When uploading a pack where a prior pack exists, would be great to have Studio ask if the user wants to reset the pack to defaults.
If pack changed authors and/or versions so the PUP files are very different where the old pup files are no longer relevant/working, Studio doesnt know.

**Comment by syd711** [comment]
"Reset" means here: delete the existing folder all files before extracting the new one, right?

**Comment by Ltek** [comment]
> "Reset" means here: delete the existing folder all files before extracting the new one, right?

Yes, delete the entire folder. Sorry, I should have said that specifically.

**Comment by syd711** [comment]
I have added a cleanup task before the actual PUP pack installation now. This way, no old corpses should remain.
An additional warning is too much effort right now as I have to go through all asset types again for this type of check.

**Comment by Ltek** [comment]
understood... basic delete-all then install just keeps it clean. In case the user customized the .pup's and PinUpPlayer.ini in the root, might be good to have a popup asking if they want to overwrite, save, or back them up?
